source/Engine.cpp

```diff
@@ -1987,7 +1987,7 @@ void Engine::SpawnFleets()
 	// or coming from planets in the current one.
 	ConditionsStore &conditions = player.Conditions();
 	for(const auto &fleet : player.GetSystem()->Fleets())
-		if(!Random::Int(fleet.Period()) && fleet.CanTrigger(conditions))
+		if(!Random::Int(fleet.Period() / GameData::GetGamerules().FleetMultiplier()) && fleet.CanTrigger(conditions))
```

Comment:
I'd rather make a special case for when the fleet multiplier is 0.
```suggestion
		if(fleetMultiplier ? !Random::Int(fleet.Period() / fleetMultiplier) && fleet.CanTrigger(conditions) : true)
```

Reply:
Wouldn't a zero logically mean no fleets at all?

Reply:
It seems to work in my testing.